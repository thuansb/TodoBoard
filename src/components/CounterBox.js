import React from 'react';

function CounterBox({ count }) {
    return (
        <div className="counter-box">
            {count}
            <div className="counter-box__text">TASKS</div>
        </div>
    )
}

export default CounterBox;