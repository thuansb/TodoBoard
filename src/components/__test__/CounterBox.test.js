import React from 'react';
import CounterBox from '../CounterBox';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <CounterBox counter={10000} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
