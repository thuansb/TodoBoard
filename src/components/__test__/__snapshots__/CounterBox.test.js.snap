// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="counter-box"
>
  <div
    className="counter-box__text"
  >
    TASKS
  </div>
</div>
`;
